docs/index.asciidoc

```diff
@@ -292,8 +292,8 @@ This plugin supports the following configuration options plus the <<plugins-{typ
 
 // Variables for re-use in per-option docs
 :prohibit-ssl-disabled-effective: Cannot be combined with configurations that disable SSL
-:prohibit-ssl-disabled-explicit: Cannot be combined with <<plugins-{type}s-{plugin}-ssl_enabled, `ssl_enabled \=> false`>>.
-:prohibit-ssl-verify-none: Cannot be combined with <<plugins-{type}s-{plugin}-ssl_verification_mode, `ssl_verification_mode \=> none`>>.
+:prohibit-ssl-disabled-explicit: Cannot be combined with `<<plugins-{type}s-{plugin}-ssl_enabled>>=>false`.
+:prohibit-ssl-verify-none: Cannot be combined with `<<plugins-{type}s-{plugin}-ssl_verification_mode>>=>none`.
```

Comment:
Reformatting to make link text literal

Reply:
This approach works, but I'm not convinced that it's best practice.  This approach doesn't save us much time or repetition, yet puts added burden on a user looking at docs in the plugin repo. 

<img width="622" alt="Screen Shot 2023-12-19 at 5 35 53 PM" src="https://github.com/elastic/logstash-filter-elastic_integration/assets/35154725/a5740d74-417a-414a-90a3-92700bd82468">

Rather than clearly stating the information with the setting, we're giving users a job to do in order to go find where we set up these variables.

Reply:
I'd like to get the initial docs published soon, and I'm willing to table this for a later discussion outside of this PR.